### ISSUE TITLE
If a RadioGroup component is the last focusable element inside a Popup component, useFocusTrap does not work properly

### ISSUE THREAD

**Issue by Victor-Meng-Workday** [Workday/canvas-kit#2365]
## 🐛 Bug Report

We have a `RadioGroup` component inside of a`Popup` component. We have enabled focus trap for the popup via `useFocusTrap`. If the `RadioGroup` component is the last focusable element in the popup and I press TAB after the focus lands on the radio group, then the focus trap would break and the focus would escape to the address bar.

## To Reproduce

1. Add a `RadioGroup` component inside of a `Popup` component and make sure it's the last focusable element inside the popup
2. Open the popup
3. Keep pressing TAB until the focus lands on the radio group
4. Press TAB again
5. Notice the focus has jumped to the address bar

## Expected Behavior

The focus should be trapped inside of the popup and not escape to the address bar since the focus trap is enabled.

## Actual Results

The focus moved to the address bar outside of the popup.

## Browser (if applicable)

If this was a visual bug, what browser did you spot it on?
N/A

## Link to repl or repo (highly encouraged)

Please provide a reference to a repo or branch to help us reproduce this bug.

### Error Output

Paste the error output here

```bash

```

or

Include a screenshot of a visual bug

**Comment by Victor-Meng-Workday** [comment]
Also want to add that I've posted this issue in the canvas-kit slack channel and Nicholas has confirmed that this is a new bug.
